script/bootstrap

```diff
@@ -7,7 +7,22 @@ set -e
 
 cd "$(dirname "$0")/.."
 
+echo "==> Installing Ruby…"
+rbenv install --skip-existing
+
 echo "==> Installing gem dependencies…"
+which bundle 2>&1 > /dev/null || {
+  gem install bundler
+}
 bundle check --path vendor/gems 2>&1 > /dev/null || {
   bundle install --path vendor/gems --quiet --without production
 }
+
+if [ "$(uname -s)" = "Darwin" ] && brew --prefix 2>&1 > /dev/null; then
+  echo "==> Installing Homebrew dependencies…"
+  brew update
+  brew tap homebrew/bundle 2>/dev/null
+  brew bundle check 2>&1 > /dev/null || {
```

Comment:
This exits with 1 if there isn't a Brewfile, so and `brew bundle` fails when it's not present:

```
$ brew bundle
/opt/boxen/homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/utils.rb:35:in `rescue in brewfile': No Brewfile found (RuntimeError)
        from /opt/boxen/homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/utils.rb:28:in `brewfile'
        from /opt/boxen/homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/commands/install.rb:4:in `run'
        from /opt/boxen/homebrew/Library/Taps/homebrew/homebrew-bundle/cmd/brew-bundle:66:in `<main>'

```